setup.py

```diff
@@ -17,57 +17,17 @@
 
 # name of Interfacing API
 APPNAME = "pyprecice"
-APPVERSION = "2.0.0a1"  # todo: should be replaced with precice.get_version() as soon as it exists , see https://github.com/precice/precice/issues/261
+APPVERSION = "2.0.0a4"  # todo: should be replaced with precice.get_version() as soon as it exists , see https://github.com/precice/precice/issues/261
```

Comment:
Let's wait until #31 is resolved. :wink:

Reply:
#31 is resolved now.